sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blob/README.md

```diff
@@ -77,22 +81,21 @@ This indicates that the Event Hubs service did not respond to an operation withi
 #### Message Size Exceeded
 
 Event data, both individual and in batches, have a maximum size allowed.  This includes the data of the event, as well as any associated metadata and system overhead.  The best approach for resolving this error is to reduce the number of events being sent in a batch or the size of data included in the message.  Because size limits are subject to change, please refer to [Azure Event Hubs quotas and limits](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-quotas) for specifics.  
-
```

Comment:
Do we really want to remove this line?

Reply:
Likely not, but we'll be removing that blurb most likely.   As Ramya pointed out above, it doesn't add value here because the processor isn't publishing.